### ISSUE TITLE
Will espanso 2.0 support older macOS versions?

### ISSUE THREAD

**Issue by rflcrz** [federico-terzi/espanso#785]
Hi, @federico-terzi!

Is there any plan to support older macOS versions?

I currently use Mojave on a 5.1 MacPro. Will espanso compile on my Mac?

Thank you! 😊

**Comment by federico-terzi** [comment]
Hey @rflcrz,

Thank you for the feedback! I haven't tested Espanso on older versions of macOS and it's unlikely that we'll be able to "officially" support many versions with our current resources. That said, have you tried running the latest Espanso release? As far as I can tell, there is no explicit minimum version, so it might be working :)

If you want to try, you can download the new App bundle from here: https://github.com/federico-terzi/espanso/releases/download/v2.0.1/Espanso-Mac-Intel.zip

Let me know if that works

Cheers :)

**Comment by federico-terzi** [comment]
Keep in mind that you will probably need to follow this process: https://espanso.org/docs/next/install/mac/#problems-you-may-experience

**Comment by federico-terzi** [comment]
@rflcrz All right, things should be good now! Without going too much into the details, the CI executed a testing step before creating the release that caused some modules to be compiled without `MACOSX_DEPLOYMENT_TARGET` being set.

Could you please try this release and see if that helps? https://github.com/federico-terzi/espanso/actions/runs/1342885240

PS: thank you for the support, both as a sponsor and tester! :)

**Comment by dajare** [comment]
@federico-terzi I know this is a closed issue, but ... I've recently started using Espanso on my main machine (Linux Mint LMDE 7), and have decided I cannot live without it on my "other" machine, an early 2015 MacBook running 10.14.6 "Mojave" (needed for 32-bit applications).

I cannot get the current version of Espanso to run on that MacBook. The "release" [linked above](https://github.com/espanso/espanso/issues/785#issuecomment-943636050) now 404s. Do you have any advice on which release I could get that would run on this old machine? I live in hope...

Thanks for your help with this, and for this terrific app!

**Comment by smeech** [comment]
It's likely you'll find the Release referred to above at https://github.com/espanso/espanso/releases?page=3 or thereabouts.

Good luck!

**Comment by dajare** [comment]
@smeech - Thanks for the link! I tried both the first "beta" in that series (2.1.4), and the actual "alpha" that references this issue (2.0.3) but, sadly, neither are working for me on this machine. Sigh! Appreciate your pointer, though.

**UPDATE**: After doing some poking around (including using some "built-in" options with Mojave that were not satisfactory), I found an old, **unsupported** app called [Quickwords](https://github.com/quickwords/quickwords/releases/tag/v0.1.2) which is also a text-expander that works just fine on Mojave. It lacks some (okay, many) of Espanso's bells-and-whistles, but it gets me 94.7% of the way there for my use case. `:)` If anyone else comes looking for this, that might help.
